### PR TITLE
Sg2265/header

### PR DESCRIPTION
**Pull request by SGupta101**
Implementing header of Carriage login 
- added import statements to more cleanly add images
- work in progress (not finished)

### CODE REVIEW

File: frontend/src/pages/Landing/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            paddingTop: 30,
+            paddingBottom: 30,
+          }}
+        >
+          Students schedule, <br />
+          edit, and cancel rides <br />
+          on rider web <br />
+          and mobile.
+        </p>
+        <img src="frontend/src/pages/Landing/Group525.png" />
```
Comment (Atikpui007) [inline frontend/src/pages/Landing/Footer.tsx:25]: Maintain consistency for import method 

use ``
Also for all your images be sure to add an alt tag and text for accessibility reasons

**Review by Atikpui007 — CHANGES_REQUESTED** [review]
Looks good so far a few things to note. 

- Inline styling isn't really recommended you can make use of the `footer.module.css` file for the Footer styles
- The Footer is currently not showing because it is conflicting with the Footer component in `../frontend/src/components/Footer/Footer.tsx`

Overall good progress

File: frontend/package.json [hunk 2]
```diff
@@ -37,7 +37,7 @@
     "typescript": "^4.8.4"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "react-scripts --openssl-legacy-provider start",
```
Comment (pratyush1712) [inline frontend/package.json:40]: can you remove this addition? it's compatible only with React version 18+ and I do not think all the developers on that version. It would be better to ask all the developers to upgrade in a separate dedicated Pull Request

File: frontend/src/pages/Landing/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
+import React, { ReactElement } from 'react';
+import styles from './landing.module.css';
+import { dti_logo } from '../../icons/other';
+import group525 from './images/Group525.png';
```
Comment (pratyush1712) [inline frontend/src/pages/Landing/Footer.tsx:4]: Please use more descriptive names for images

**Review by pratyush1712 — COMMENTED** [review]
The changes look pretty good! Please address the issues I have mentioned and it should be good.
